dll/win32/iernonce/include/registry.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Functions to read RunOnceEx registry.
+ * COPYRIGHT:   Copyright 2021 He Yang <1160386205@qq.com>
+ */
+
+
+#include <windows.h>
+#include <set>
+#include <map>
+#include <atlbase.h>
+#include <atlstr.h>
+#include <atlcoll.h>
+
+
+typedef class RunOnceExSection *pRunOnceExSection;
+
+
+class CRegKeyEx : public CRegKey
+{
+public:
+    LONG EnumValueName(
+        DWORD iIndex,
+        LPTSTR pszName,
+        LPDWORD pnNameLength);
+};
+
+// for sorting stuffs in STL container.
+struct RunOnceExEntryCmp {
+    bool operator()(const ATL::CStringW &lhs, const ATL::CStringW &rhs) const;
+};
+struct RunOnceExSectionCmp {
+    bool operator()(const pRunOnceExSection &lhs, const pRunOnceExSection &rhs) const;
+};
+
+typedef std::map<ATL::CStringW, ATL::CStringW, RunOnceExEntryCmp> RUNONCE_ENRTY_MAP;
```

Comment:
```suggestion
typedef std::map<ATL::CStringW, ATL::CStringW, RunOnceExEntryCmp> RUNONCE_ENTRY_MAP;
```

Reply:
ah... is there any difference in the suggested change? 😶

Reply:
the typo in "ENTRY"? (github diff clearly overlines what word is different there)